src/dataBaseServices/firebaseDBServices.js

```diff
@@ -0,0 +1,36 @@
+import { getDatabase, ref, onValue, set } from 'firebase/database';
+
+class IFirebaseDBServices {
+  getTasks() {
+    throw new Error('You need to implement this method');
+  }
+
+  setTasks() {
+    throw new Error('You need to implement this method');
+  }
+}
+
+export class FirebaseDBServices extends IFirebaseDBServices {
```

Comment:
Почему множественное число ?
```suggestion
export class FirebaseDBService extends IFirebaseDBServices {
```